tests/test_jsonpointer.py

```diff
@@ -123,11 +123,19 @@ def test_compare_jsonpointer(value1, value2):
 @given(relative_jsonpointer)
 def test_create_relative_jsonpointer(value):
     match = re.fullmatch(relative_jsonpointer_regex, value)
-    up, ref = match.group('up', 'ref')
+    up, over, ref = match.group('up', 'over', 'ref')
+    kwargs = {
+        'up': up,
+        'ref': ref,
+    }
+    if over:
+        kwargs['over'] = over
```

Comment:
```py
    kwargs = {
        'up': up,
        'over': over,
        'ref': ref,
    }
```
suffices here, since we're handling `''` as if it were `0`.